pkgs/tools/X11/xdg-utils/default.nix

```
@@ -30,6 +32,8 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-8PtXfI8hRneEpnUvIV3M+6ACjlkx0w/NEiJFdGbbHnQ=";
   };
 
+  patches = lib.optional withPortalPatch ./use-portal.patch;
```

Probably better to indicate that this patch enables portals to work in FHSEnvs. I.e. `fhs-use-portal.patch`.

---

Comment would be useful too, see https://github.com/jtojnar/nixpkgs-hammering/blob/277c829804ba83b590dea10f04b39a8fffba7d6d/explanations/missing-patch-comment.md.

```suggestion
  patches = lib.optionals withPortalPatch [
    # Allow forcing the use of XDG portals using XDG_OPEN_USE_PORTAL environment variable.
    ./allow-forcing-portal-use.patch
  ];
```

And `optionals` allows for more consistent formatting when other patches are added.

---

Portal also works outside of FHS eg with wrappers setting LD_LIBRARY_PATH so not sure about renaming it.

---

Added comment and linked to upstream PR, hope it looks good now!